scripts/roles.json

```diff
@@ -4430,116 +5237,119 @@
-  },
-  "switch": {
-    "superClass": ["checkbox"],
-    "accessibleNameRequired": true,
-    "nameFrom": ["author", "contents"],
+    "relatedConcepts": [
+      {
+        "concept": {
+          "name": "button"
+        },
+        "module": "ARIA"
+      }
+    ],
     "requiredContextRole": [],
     "requiredOwnedElements": [],
-    "requiredProps": [
-      ["aria-checked", "false"]
-    ],
+    "requiredProps": ["aria-checked"],
+    "superClass": ["checkbox"]
+  },
+  "tab": {
+    "abstract": false,
+    "accessibleNameRequired": false,
+    "childrenPresentational": true,
+    "nameFrom": ["author", "contents"],
+    "prohibitedProps": [],
     "props": [
-      ["aria-checked", "false"],
       "aria-atomic",
       "aria-busy",
       "aria-controls",
+      "aria-current",
       "aria-describedby",
+      "aria-details",
       "aria-disabled",
       "aria-dropeffect",
+      "aria-expanded",
       "aria-flowto",
       "aria-grabbed",
       "aria-haspopup",
       "aria-hidden",
-      "aria-invalid",
+      "aria-keyshortcuts",
       "aria-label",
       "aria-labelledby",
       "aria-live",
       "aria-owns",
-      "aria-relevant"
-    ],
-    "abstract": false,
-    "childrenPresentational": true,
-    "relatedConcepts": [{
-      "module": "ARIA",
-      "concept": {
-        "name": "button"
-      }
-    }]
-  },
-  "tab": {
-    "superClass": ["sectionhead", "widget"],
-    "accessibleNameRequired": false,
-    "nameFrom": ["author", "contents"],
-    "requiredContextRole": [
-      "tablist"
+      "aria-posinset",
+      "aria-relevant",
+      "aria-roledescription",
+      "aria-setsize",
+      ["aria-selected", "false"]
     ],
+    "relatedConcepts": [],
+    "requiredContextRole": ["tablist"],
     "requiredOwnedElements": [],
     "requiredProps": [],
+    "superClass": ["sectionhead", "widget"]
+  },
+  "table": {
     "abstract": false,
-    "relatedConcepts": [],
+    "accessibleNameRequired": true,
+    "childrenPresentational": false,
+    "nameFrom": ["author"],
+    "prohibitedProps": [],
     "props": [
-      "aria-posinset",
-      ["aria-selected", "false"],
-      "aria-setsize",
-      "aria-expanded",
       "aria-atomic",
       "aria-busy",
+      "aria-colcount",
       "aria-controls",
+      "aria-current",
       "aria-describedby",
-      "aria-disabled",
+      "aria-details",
       "aria-dropeffect",
       "aria-flowto",
       "aria-grabbed",
-      "aria-haspopup",
       "aria-hidden",
-      "aria-invalid",
+      "aria-keyshortcuts",
       "aria-label",
       "aria-labelledby",
       "aria-live",
       "aria-owns",
-      "aria-relevant"
+      "aria-relevant",
+      "aria-roledescription",
+      "aria-rowcount"
     ],
-    "childrenPresentational": true
-  },
-  "table": {
-    "superClass": ["section"],
-    "accessibleNameRequired": true,
-    "nameFrom": ["author"],
-    "requiredContextRole": [],
-    "requiredOwnedElements": [
-      ["row"],
```

Comment:
There are several examples like this change where arrays are removed, creating mixed content inside `requiredOwnedElements`. Could you revert these changes so that `requiredOwnedElements` consistently contains arrays?

Reply:
Done